src/aura/AuraAvatarMultiToken.sol

```diff
@@ -487,22 +448,43 @@ contract AuraAvatarTwoToken is BaseAvatar, PausableUpgradeable, AuraAvatarUtils,
         AURA.safeTransfer(ownerCached, totalAura);
     }
 
+    /// @dev given a target PID, it will add the details in the `EnumerableSet`: pids, assets & baseRewardPools
+    /// @param _newPid target pid numeric value to add in contract's storage
+    function addBptPositionInfo(uint256 _newPid) external onlyOwner {
+        pids.add(_newPid);
+        (address lpToken,,, address crvRewards,,) = AURA_BOOSTER.poolInfo(_newPid);
+        assets.add(lpToken);
+        baseRewardPools.add(crvRewards);
+    }
+
+    /// @dev given a target PID, it will remove the details from the `EnumerableSet`: pids, assets & baseRewardPools
+    /// @param _removePid target pid numeric value to remove from contract's storage
+    function removeBptPositionInfo(uint256 _removePid) external onlyOwner {
+        require(pids.contains(_removePid), "AuraAvatarMultiToken: PID doesnt exist!");
+
+        (address lpToken,,, address crvRewards,,) = AURA_BOOSTER.poolInfo(_removePid);
+        uint256 stakedBal = IBaseRewardPool(crvRewards).balanceOf(address(this));
```

Comment:
```suggestion
        uint256 stakedAmount = IBaseRewardPool(crvRewards).balanceOf(address(this));
```

Reply:
renamed in 1bf3520289ab8590692162543359a3039d54602b